Assets/PoisonMushroom.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PoisonMushroom : MonoBehaviour
+{
+    //Mushroom stats
+    public int health; //Mushroom health
```

Comment:
the health variable should be removed since it already exists in the parent class EnvironmentHealth